GestaoFinanceira/GestaoFinanceira/Program.cs

```diff
@@ -16,7 +16,7 @@ static void Main()
         {
             Application.EnableVisualStyles();
             Application.SetCompatibleTextRenderingDefault(false);
-            Application.Run(new Form1());
+            Application.Run(new frm_DashBoard());
```

Comment:
@MessiasOli , é necessário alterar os nomes da classe dentro do arquivo

```suggestion
           Application.Run(new FrmDashBoard());
```